person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, name = 'unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] Please 🙏🏽 add a method for adding rentals should be present. Link to the example in a [previous lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship). The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association.